msbuild/Xamarin.iOS.Tasks.Core/Tasks/ResolveNativeWatchAppTaskBase.cs

```diff
@@ -55,6 +55,10 @@ public override bool Execute ()
 				version = currentSdk.GetClosestInstalledSdk (version, false);
 				sdk_path = currentSdk.GetSdkPath (version, SdkIsSimulator);
 			} else {
+				if (AppleSdkSettings.XcodeVersion.Major >= 10) {
+					Log.LogError ("Xcode 10 does not support watchOS 1 apps. Either upgrade to watchOS 2 apps, or use an earlier version of Xcode.");
+					return false;
```

Comment:
Maybe instead of saying "earlier version", say "older version". I think that might read better in English. At least *I* usually say "older version".